test/helper/testHelpers.ts

```diff
@@ -1,4 +1,6 @@
 import { BN } from '@flarenetwork/flarejs/dist';
+import { exportTxCP } from '../../src/evmAtomicTx';
+import { contextEnv } from '../../src/constants';
 export const serialize = (bufferData: Buffer[]) => {
```

Comment:
add blank line above
please add documentation for all the exported function, interfaces, constants and enums